packages/manager/src/features/LoadBalancers/LoadBalancerLanding/LoadBalancerActionsMenu.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { Action, ActionMenu } from 'src/components/ActionMenu';
+
+interface Props {
+  loadBalancerId: number;
+}
+
+export const LoadBalancerActionsMenu = (props: Props) => {
+  const { loadBalancerId } = props;
+
+  const actions: Action[] = [
+    {
+      onClick: () => null,
```

Comment:
Can we add the linking? 

```suggestion
      onClick: () => history.push(`/loadbalancers/${loadBalancerId}/configurations`),
```